inference-engine/src/mkldnn_plugin/nodes/mkldnn_roi_pooling_node.cpp

```diff
@@ -358,6 +426,12 @@ void MKLDNNROIPoolingNode::createPrimitive() {
 
     jpp.nb_c_blocking = mayiuse(cpu::x64::avx512_common) ? 15 : 7;
 
+    auto selectedPD = getSelectedPrimitiveDescriptor();
+    jpp.src_prc = selectedPD->getConfig().inConfs[0].desc.getPrecision();
+    jpp.dst_prc = selectedPD->getConfig().outConfs[0].desc.getPrecision();
+    jpp.src_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jpp.src_prc));
+    jpp.dst_data_size = MKLDNNExtensionUtils::sizeOfDataType(MKLDNNExtensionUtils::IEPrecisionToDataType(jpp.dst_prc));
```

Comment:
`InefernceEngine::Precision` class has `size()` method, let's avoid the extra temporal precision to data type conversion.

Reply:
Corrected